daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/SchemaDocument.scala

```diff
@@ -261,9 +262,16 @@ final class SchemaDocument private (xmlSDoc: XMLSchemaDocument)
   lazy val defineVariables = annotationObjs.collect { case dv: DFDLDefineVariable => dv }
 
   /**
-   * by name getters for the global things that can be referenced.
+   * by name/qname getters for the global things that can be referenced.
    */
-  def getGlobalElementDecl(name: String) = globalElementDecls.find { _.name == name }
+  def getGlobalElementDecl(qname: RefQName): Option[GlobalElementDecl] =
```

Comment:
Same question as before, why is this RefQName when the other similar functions are just a String?